src/main/java/org/jabref/model/entry/Month.java

```diff
@@ -110,7 +110,7 @@
      * @return the corresponding month instance, empty if input is not in German
      *         form
      */
-    private static Optional<Month> parseGermanShortMonth(String value) {
+    public static Optional<Month> parseGermanShortMonth(String value) {
```

Comment:
Maybe just remove the public prefix and make it "default"

Reply:
I'm not sure I understand it correctly, you mean like this: [5022f10](https://github.com/JabRef/jabref/pull/7542/commits/5022f103f355d09b55cfff8ca5219772d883e030)?

Reply:
Yes, this makes it package visible, so it can be accessed by the unit test but not from others https://www.baeldung.com/java-access-modifiers